src/Filament/Resources/PageResource.php

```diff
@@ -0,0 +1,123 @@
+<?php
```

Comment:
Suggest we make all of these `abstract`, e.g `src/Filament/Resources/AbstractContentResource.php` etc.  Then create `src/Filament/Resources/PageResource.php` extending `AbstractContentResource`, not just `Resource`.